### PR TITLE
Minor fixes.

### PR DESCRIPTION
**Pull request by sinexu**

### CODE REVIEW

File: setuptool/setuptool.go [hunk 1]
```diff
@@ -32,8 +32,17 @@ func main() {
 	flag.Parse()
 	fmt.Printf("Welcome to the Web Api Gateway Config Tool.\n\n")
 
+	c, _, err := config.ReadWriteConfig()
+	if err != nil {
+		fmt.Printf("Unable to load config file %v\n", err)
```
Comment (imangoodarzi) [inline setuptool/setuptool.go:37]: should you return and not continue if there is error?

File: setuptool/setuptool.go [hunk 2]
```diff
@@ -324,12 +333,12 @@ func retrieveAccountKey(t *term) {
 
 func addUser(t *term) {
 	userInput(t, "Enter the users' emails> ", config.AddUser)
-	fmt.Println("User added.")
+	fmt.Println("User added.\n")
```
Comment (imangoodarzi) [inline setuptool/setuptool.go:337]: why do you need \n when you are doing println? Do you want extra empty line between the outputs?

**Review by imangoodarzi — APPROVED** [review]

File: setuptool/setuptool.go (hunk 2 above)
Comment (sinexu) [reply setuptool/setuptool.go:337]: Yep one more line seems clearer=)

File: setuptool/setuptool.go (hunk 1 above)
Comment (sinexu) [reply setuptool/setuptool.go:37]: Thanks! This should never happen, so i would just go with os.Exit(1）

**Pull request merged**